etc/segments.csv

```diff
@@ -0,0 +1,64 @@
+SEGMENT,REPLACEMENT
+(,
+),
+[,
+],
+?,ʔ
+',ʔ
+'),ʔ
+'a,ʔ a
+'e,ʔ e
+'i,ʔ i
+'í,ʔ í/i
+'o,ʔ o
+'u,ʔ u
+(h,h
+)h,h
+∫,ʃ
+á:,á:/aː
+á,á/a
+à,à/a
+ch,ʃ
+čh,tʃʰ
+é:,é:/eː
+é,é/e
+è,è/e
+ê,ê/e
+éː,éː/eː
+éh,é/e h
+hn,ʰn
```

Comment:
So if `hn` is pre-aspirated, how come `hr` is not preaspirated as well? or `hw`? And how come `hr` is `hɾ` and `hɾ` is `h ɾ`?

Reply:
They are indeed, the reported quasi-Spanish original ortography is clearer.